src/global/Flags.h

```diff
@@ -179,23 +179,23 @@ class NODISCARD Flags
     }
 
 public:
-    NODISCARD inline bool contains(const Flag flag) const noexcept
+    NODISCARD inline constexpr bool contains(const Flag flag) const noexcept
```

Comment:
These constexpr enhancements should also be a separate PR

Reply:
I did this

Reply:
shall I revert these constexpr enhancements?